modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -72,6 +114,22 @@ definition {
 		Click(locator1 = "AppBuilder#SAVE_BUTTON");
 	}
 
+	macro selectForm {
```

Comment:
Hey Brooke, could you please rename this macro to selectFormView? so we can keep the pattern of the others macros.